### PR TITLE
Fix some L10N regressions

### PR DESCRIPTION
**Pull request by jasonLaster**
Associated Issue: #1783, #1784

### Summary of Changes

* Fix key collisions
* Fix %s 

### Test Plan

- [ ] View search boxes
- [ ] View command tooltips

CC @flodolo 

@clarkbw r?

### CODE REVIEW

File: assets/locales/debugger.properties [hunk 1]
```diff
@@ -116,15 +116,15 @@ blackBoxCheckboxTooltip=Toggle black boxing
...
+sourceSearch.search1.key=F
 
-# LOCALIZATION NOTE (sourceSearch.search.again.key): Key shortcut to re-open
+# LOCALIZATION NOTE (sourceSearch.search1.again.key): Key shortcut to re-open
 # the search for re-searching the same search triggered from a sourceSearch
-sourceSearch.search.again.key=G
+sourceSearch.search1.again.key=G
 
-# LOCALIZATION NOTE (sourceSearch.search.resultsSummary): Shows a summary of
+# LOCALIZATION NOTE (sourceSearch.search1.resultsSummary): Shows a summary of
```
Comment (flodolo) [inline assets/locales/debugger.properties]: Reference to string ID is wrong here

File: assets/locales/debugger.properties [hunk 2]
```diff
@@ -116,15 +116,15 @@ blackBoxCheckboxTooltip=Toggle black boxing
 # searching all the source files the debugger has seen.
 sources.search.key=P
 
-# LOCALIZATION NOTE (sourceSearch.search.key): Key shortcut to open the search
+# LOCALIZATION NOTE (sourceSearch.search1.key): Key shortcut to open the search
 # for searching within a the currently opened files in the editor
-sourceSearch.search.key=F
+sourceSearch.search1.key=F
```
Comment (flodolo) [inline assets/locales/debugger.properties]: No need to change this one. The string didn't exist, it's a shortcut so it doesn't depend on the label.

File: assets/locales/debugger.properties [hunk 3]
```diff
@@ -116,15 +116,15 @@ blackBoxCheckboxTooltip=Toggle black boxing
...
 
-# LOCALIZATION NOTE (sourceSearch.search.again.key): Key shortcut to re-open
+# LOCALIZATION NOTE (sourceSearch.search1.again.key): Key shortcut to re-open
 # the search for re-searching the same search triggered from a sourceSearch
-sourceSearch.search.again.key=G
+sourceSearch.search1.again.key=G
```
Comment (flodolo) [inline assets/locales/debugger.properties]: Same here, no need to change the string ID.

File: assets/locales/debugger.properties [hunk 4]
```diff
@@ -253,7 +253,7 @@ editor.conditionalPanel.placeholder=This breakpoint will pause when the expressi
 
 # LOCALIZATION NOTE(editor.jumpToMappedLocation): Context menu item
 # for navigating to a source mapped location
-editor.jumpToMappedLocation=Jump to %s location
+editor.jumpToMappedLocation=Jump to %S location
```
Comment (flodolo) [inline assets/locales/debugger.properties]: In this case you need a new string ID, the variable changes, dashboard will throw errors.

**Comment by jasonLaster** [comment]
We should update the description for block

>> - Block is a noun. I can update the comment to explain that we're referring
> to a block of code.

**Pull request merged**